### DISCUSSION TITLE
Locales from back-end

### DISCUSSION THREAD

**Discussion by GTOsss** [vercel/next.js#22859] [Ideas]
### Describe the feature you'd like to request

I read the documentation on localization implementation strategies, but I didn't see the possibility of loading the localization file asynchronously. In my case, the possible locales are stored on the backend and I can't know about them when configuring `next.config`
Is it possible to initialize and add new locales in runtime? I mean here's an opportunity like this:
https://www.i18next.com/how-to/add-or-load-translations#add-after-init

### Describe the solution you'd like

I would like to have a function that will allow you to add a possible locale after starting the next server.

### Describe alternatives you've considered

As an alternative, it remains to implement only GSSP which will unload the list of possible locales, but this does not solve many problems: 
- automatic locale detection 
- routing
- and possibly some others

**Comment by SemenovYar** [comment]
same issue

**Comment by sdoomz** [comment]
have the same issue
